### PR TITLE
Add Katello triage documentation

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
Translated from an email I sent to -dev. 

@jlsherrill

File: plugins/katello/developers.md [hunk 1]
```diff
@@ -26,6 +26,70 @@ The Katello bug triage process is designed to address Redmine issues that have b
   * [Untriaged Open](http://projects.theforeman.org/projects/katello/issues?utf8=%E2%9C%93&set_filter=1&f%5B%5D=release_id&op%5Brelease_id%5D=%21*&f%5B%5D=subproject_id&op%5Bsubproject_id%5D=%21*&f%5B%5D=status_id&op%5Bstatus_id%5D=o&f%5B%5D=&c%5B%5D=tracker&c%5B%5D=status&c%5B%5D=priority&c%5B%5D=subject&c%5B%5D=author&c%5B%5D=assigned_to&c%5B%5D=updated_on&c%5B%5D=category&c%5B%5D=fixed_version&group_by=)
   * [Untriaged Ready for Testing](http://projects.theforeman.org/projects/katello/issues?utf8=%E2%9C%93&set_filter=1&f%5B%5D=release_id&op%5Brelease_id%5D=%21*&f%5B%5D=subproject_id&op%5Bsubproject_id%5D=%21*&f%5B%5D=status_id&op%5Bstatus_id%5D=%3D&v%5Bstatus_id%5D%5B%5D=7&f%5B%5D=&c%5B%5D=tracker&c%5B%5D=status&c%5B%5D=priority&c%5B%5D=subject&c%5B%5D=author&c%5B%5D=assigned_to&c%5B%5D=updated_on&c%5B%5D=category&c%5B%5D=fixed_version&group_by=)
 
+### Triage Runner
+
+We have a few standing developers that switch off running triage. Typically it is myself or Justin but we try to get the current release owner to jump in and run it as well as encouraging others to take it on. Takes one or two to get the hang of it, but it is a good experience.
```
Comment (akofink) [inline plugins/katello/developers.md]: I'd reword this paragraph, remove first person (myself).

File: plugins/katello/developers.md [hunk 2]
```diff
@@ -26,6 +26,70 @@ The Katello bug triage process is designed to address Redmine issues that have b
...
+
+  * Empty: No release has been set, this is the "untriaged" bucket
+  * Katello Backlog: Issues that have been put on the backlog and not scheduled for a release due to any number of reasons
+  * Katello Recycle Bin: Issues that have been rejected, or are a duplicate, or have been in 'Needs more information' for longer than a week
+  * Katello X.Y.Z: Standard releases for issues to be targeted at
+
+### Process
+
+The triage runner starts the meeting by going through sets of issues broken down into different groups by priority of the grouping. This is so that if time were to run out, the most critical categories of issues are triaged. This list (with easy links for the runner) are at [1]. The runner is in charge of ensuring the flow of the meeting, and updating issues or ensuring that someone on the call takes responsibility for updating an issue. This latter part sometimes occurs if a duplicate needs to be looked up, or another developer on the call can explain/ask the user for more information more clearly than the runner.
```
Comment (akofink) [inline plugins/katello/developers.md]: Could we include the triage issue lists link inline?

File: plugins/katello/developers.md [hunk 3]
```diff
@@ -26,6 +26,70 @@ The Katello bug triage process is designed to address Redmine issues that have b
...
+
+### Recycle Bin
+
+The first on the list is our 'recycle bin'. This the place to put issues that don't fit into a release, or were filed by a user who has since gone dark on the issue. Another way to look at it is a place for unresolvable issues. Issues are moved onto the recycle bin if there has been no activity by a user for a week. And they are closed if a week on the recycle bin goes by without any further updates.
+
+#### General Flow for New or Needs More Information
+
+The general flow is to open an issue and the runner to exam the issue, typically reading the general idea outloud. This allows for any developers with knowledge to chime in with any relevant info. The runner attempts to answers the following questions:
```
Comment (akofink) [inline plugins/katello/developers.md]: s/outloud/out loud/

File: plugins/katello/developers.md [hunk 4]
```diff
@@ -26,6 +26,70 @@ The Katello bug triage process is designed to address Redmine issues that have b
...
+
+	* Does this issue need more information from the user?
+	* If yes, leave a comment asking for that information
+	* Does this issue need a developer to investigate it further to properly triage it?
```
Comment (akofink) [inline plugins/katello/developers.md]: If yes, set status to Needs Design?

File: plugins/katello/developers.md [hunk 5]
```diff
@@ -26,6 +26,70 @@ The Katello bug triage process is designed to address Redmine issues that have b
...
+	* Issue category?
+
+#### General Flow for Ready for Testing
+
+  * Will this PR get completed by the next release?
+  * What release should the issue be targeted for? (e.g. next z-stream, current release, next release, backlog)
+
+#### General Flow for Closed
+
+  * Should this issue be targeted at the next z-stream? Current RC phase? Or just throw it in the "next" release (aka develop or master)
```
Comment (akofink) [inline plugins/katello/developers.md:85]: This is a general flow for any state, it seems. Perhaps we could just say this at the end of this list.

**Review by akofink — COMMENTED** [review]
Comments inline. Thanks for updating this @ehelms!

File: plugins/katello/developers.md (hunk 2 above)
Comment (ehelms) [reply plugins/katello/developers.md]: Only reason I didn't was so that the person running the meeting can quickly get to the links.

File: plugins/katello/developers.md (hunk 5 above)
Comment (ehelms) [reply plugins/katello/developers.md:85]: I thought about that, but I personally liked thinking about each state respectively.

**Comment by ehelms** [comment]
Thanks for the review, I incorporated suggestions and replied with some comments on my thinking.

File: plugins/katello/developers.md [hunk 6]
```diff
@@ -26,6 +26,71 @@ The Katello bug triage process is designed to address Redmine issues that have b
...
+The first on the list is our 'recycle bin'. This the place to put issues that don't fit into a release, or were filed by a user who has since gone dark on the issue. Another way to look at it is a place for unresolvable issues. Issues are moved onto the recycle bin if there has been no activity by a user for a week. And they are closed if a week on the recycle bin goes by without any further updates.
+
+#### General Flow for New or Needs More Information
+
+The general flow is to open an issue and the runner to exam the issue, typically reading the general idea aloud. This allows for any developers with knowledge to chime in with any relevant info. The runner attempts to answers the following questions:
+
+	* Does this issue need more information from the user?
+	  * If yes, leave a comment asking for that information
+	* Does this issue need a developer to investigate it further to properly triage it?
+    * If yes, set status to 'Needs Design'
```
Comment (akofink) [inline plugins/katello/developers.md]: Spacing looks off here.

File: plugins/katello/developers.md (hunk 2 above)
Comment (akofink) [reply plugins/katello/developers.md]: I mean, what is [1]?

Comment (ehelms) [reply plugins/katello/developers.md]: Ohhh, yea, let me remove that since this was copied from an email.

**Comment by ehelms** [comment]
Updated the indented list

**Review by akofink — APPROVED** [review]
LGTM! ACK!

**Review by dLobatog — APPROVED** [review]
I'm going to go with merging since it's already approved by @akofink & there are no obvious typos. Interesting info, especially the recycle bin sounds like a great idea to me. Thanks @ehelms
